recipe/meta.yaml

```
@@ -18,18 +18,24 @@ build:
   script: CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
   # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
   skip: true  # [osx]
```

This skip condition needs an update with the presence of CUDA (since cuQuantum doesn't support Windows), let's refine it:
```suggestion
  skip: true  # [not linux64]
```
We may need to further distinguish CUDA 11/12, but it can be done at a later time when the bot sends a migration PR to this feedstock.

---

(updated the skip condition a bit)

---

Note that while this is effectively done at the dependency level, we still need to be explicit here so as to reduce the number of Windows CIs (see the excessive number of CI pipelines launched).